{{cookiecutter.project_shortname}}/docker/wait-for-services.sh

```diff
@@ -27,6 +27,9 @@ check_ready "MySQL" _db_check
 _es_check(){ curl --output /dev/null --silent --head --fail http://localhost:9200 &>/dev/null; }
 check_ready "Elasticsearch" _es_check
 
+_kibana_check(){ curl --output /dev/null --silent --head --fail http://localhost:5601 &>/dev/null; }
```

Comment:
Hm, we should either not check for `kibana` (since it's a monitoring tool) or also check for `flower` for consistency.